plugins/apache_http.yaml

```diff
@@ -0,0 +1,82 @@
+version: 0.0.1
+title: Apache HTTP Server
+description: Log parser for Apache HTTP Server
+parameters:
+  enable_error_log:
+    label: Error Logs
+    description: Enable to collect Apache HTTP Server error logs
+    type: bool
+    default: true
+  error_log_path:
+    label: Error Log Path
+    description: Path to error log file
+    type: string
+    default: "/var/log/apache2/error.log"
+  enable_access_log:
+    label: Access Logs
+    description: Enable to collect Apache HTTP Server access logs
+    type: bool
+    default: true
+  access_log_path:
+    label: Access Log Path
+    description: Path to access log file
+    type: string
+    default: "/var/log/apache2/access.log"
+  start_at:
+    label: Start At
+    description: Start reading file from 'beginning' or 'end'
+    type: enum
+    valid_values:
+     - beginning
+     - end
+    default: end
+
+  # Set Defaults
```

Comment:
I'd fix the indentation here. It's not a big deal, but it would help this plugin conform with the others. Also, add a new line between the last default set and the pipeline.